src/components/Toolbelt/Toolbelt.js

```diff
@@ -5,68 +5,98 @@ import classNames from 'classnames'
 import Button from '@material-ui/core/Button'
 import Tooltip from '@material-ui/core/Tooltip'
 
+import { toolLevel } from '../../enums'
+
 import { memoize } from '../../utils'
 
 import FarmhandContext from '../../Farmhand.context'
 import toolsData from '../../data/tools'
 
-import { tools as toolImages, pixel } from '../../img'
+import { tools as toolImages, craftedItems, pixel } from '../../img'
 
 import './Toolbelt.sass'
 
 const noop = () => {}
-const getTools = memoize(shovelUnlocked => {
-  return Object.values(toolsData)
-    .filter(t => shovelUnlocked || t.id !== 'shovel')
-    .sort(t => t.order)
+
+const getTools = memoize(toolLevels => {
+  let tools = []
+
+  for (let tool of Object.values(toolsData)) {
+    if (toolLevels[tool.type]) {
+      tools.push(tool)
+    }
+  }
+
+  tools.sort(t => t.order)
+
+  return tools
 })
 
+const getToolImage = tool => {
+  if (tool.level === toolLevel.DEFAULT) {
+    return toolImages[tool.id]
+  }
+
+  let id = `${tool.id}-${tool.level.toLowerCase()}`
```

Comment:
nit: Please change this `let` to `const` (or use the value directly in the lookup below).